test/test_autograd.py

```diff
@@ -4533,9 +4533,9 @@ def test_grad_fn_attr_bindings(self):
         out = torch.logit(a)
         self.assertIsNone(out.grad_fn._saved_eps)
 
-        a = torch.tensor([1.], requires_grad=True)
-        out = torch.div(a, 2., rounding_mode="trunc")
-        self.assertEqual(out.grad_fn._saved_rounding_mode, "trunc")       # std::string -> str
+        a = torch.ones(1, 1, requires_grad=True)
```

Comment:
Why change this to be a linalg.qr test?

Reply:
Because it doesn't have a `str` argument any more. See #55225 where I add it back.